README.md

```diff
@@ -2,12 +2,118 @@
 
 ![License](https://img.shields.io/badge/license-Apache2.0-green) ![Language](https://img.shields.io/badge/language-C++-blue.svg) [![Version](https://img.shields.io/github/v/tag/opengemini/opengemini-client-cpp?label=release&color=blue)](https://github.com/opengemini/opengemini-client-cpp/releases)
 
+> ⚠️  **This repository is in pre-alpha stage, so not yet feature-complete. And the API has not been frozen, it may be changed on the course of the development.**
 
 English | [简体中文](README_CN.md)
 
 `opengemini-client-cpp` is a C++ client for OpenGemini
 
-> ⚠️  **This repository is in pre-alpha stage, so not yet feature-complete. And the API has not been frozen, it may be changed on the course of the development.**
+---
+
+- [Prerequisites](#prerequisites)
```

Comment:
good toc part, I think we'd better adapt this toc part to other repos @Chenxulin97